unified-codes/apps/data-loader/src/app/v2/DataParser.ts

```diff
@@ -162,32 +156,23 @@ export class DataParser {
 
       // Parse entity graph.
       this.data.forEach((row) => {
+        const productDefinition = [
+          { name: row.product, code: row.uc2, type: EEntityType.Product },
+          { name: row.route, code: row.uc3, type: EEntityType.Route },
+          { name: row.dose_form, code: row.uc4, type: EEntityType.Form },
+          { name: row.dose_qualification, code: row.uc5, type: EEntityType.FormQualifier },
+          { name: row.strength, code: row.uc6, type: EEntityType.DoseStrength },
+          { name: row.unit_of_presentation, code: row.uc7, type: EEntityType.Unit },
+          { name: row.immediate_packaging, code: row.uc8, type: EEntityType.PackImmediate },
+          { name: row.pack_size, code: row.uc9, type: EEntityType.PackSize },
+          // { name: row.outer_packaging, code: row.uc10, type: EEntityType.PackOuter },
+          // { name: row.manufacturer, code: row.uc11, type: EEntityType.Manufacturer },
+          // { name: row.brand, code: row.uc12, type: EEntityType.Brand },
+        ];
+
```

Comment:
From lines 159:211 ( I can't select all in the PR.. you could make that all a bit neater now?
```suggestion
        const { uc1, combination } = row;
        const productDefinition = [
          { name: row.product, code: row.uc2, type: EEntityType.Product },
          { name: row.route, code: row.uc3, type: EEntityType.Route },
          { name: row.dose_form, code: row.uc4, type: EEntityType.Form },
          { name: row.dose_qualification, code: row.uc5, type: EEntityType.FormQualifier },
          { name: row.strength, code: row.uc6, type: EEntityType.DoseStrength },
          { name: row.unit_of_presentation, code: row.uc7, type: EEntityType.Unit },
          { name: row.immediate_packaging, code: row.uc8, type: EEntityType.PackImmediate },
          { name: row.pack_size, code: row.uc9, type: EEntityType.PackSize },
          // { name: row.outer_packaging, code: row.uc10, type: EEntityType.PackOuter },
          // { name: row.manufacturer, code: row.uc11, type: EEntityType.Manufacturer },
          // { name: row.brand, code: row.uc12, type: EEntityType.Brand },
        ];
        const productProperties: IPropertyNode[] = [
          { type: EPropertyType.RxNav, value: row.rxnav },
          { type: EPropertyType.WHOEML, value: row.who_eml_product },
          { type: EPropertyType.NZULM, value: row.nzulm },
          { type: EPropertyType.UNSPSC, value: row.unspsc },
        ];
        const itemProperties: IPropertyNode[] = [
          { type: EPropertyType.WHOEML, value: row.who_eml_item },
          { type: EPropertyType.NZULM, value: row.nzulm_item },
        ];
```

Reply:
🧹  a6724bba6